### ISSUE TITLE
Show significance of friction during ollie 

### ISSUE THREAD

**Issue by jtheinen** [mechmotum/ollie-optimization#42]
Reviewer 1 asked if the 'dragging up motion' contributes significantly and whether we could show this with our simulation. 

We have multiple options:
1. Show already present simulation of the pennyboard without friction, he can see that the ollie will be lower
2. Solve base optimization without friction and show the difference.

original text by reviewer 1:

You wrote ["for the human to ... then slide the front foot over the deck to drag it up"](https://github.com/mechmotum/ollie-optimization/blob/4d4e603d48c2a35e80c1d6a072766645e2bfddee/paper/ollie-optimization.tex#L475) at the Discussion section 8th line.
This corresponds to the timing of t5 in Fig. 1.
Do you think how much the effect of this dragging up motion is? I mean, the contribution of the friction force between the board and fore foot to the Ollie height at the timing of t5.
Can you show the effect in your simulation?
I asked it since for example the Ollie jump was realized without this dragging up motion in the simulation of the reference [21].
Basically, the force normal to the board from the fore foot at t5 is considered to be small, so the tangential friction force will become very small as well.
Therefore, the effect of dragging up motion of the fore foot might be negligibly small.
Please add some text about this issue in the revised manuscript.

**Comment by moorepants** [comment]
I don't think we show the friction force in the time series figures, right? Just adding that line to the figure would show that friction is used and when it is used.

**Comment by moorepants** [comment]
If the F abduction F(ront)/B(ack) are non-zero, that means that friction is occurring between the foot an  board? Or is it that there has to be extension causing the normal force and the abduction needs to occur? It isn't so clear.

Showing normal force and friction force on the board for both front and back, could be clearer than showing extension/abduction.

**Comment by jtheinen** [comment]
If we do not make any claims on abduction and extension forces, I agree that changing the format to the body fixed forces (normal and friction) would be more informative.

If we do make claims about abduction and extension, we might want to show them both. 

Normal forces and friction forces shown in a plot are usually overlapping (with a factor mu) so putting them both in a plot could result to unclear figures. We might want to consider just adding friction forces, since they are unclear from the current plots.

**Comment by moorepants** [comment]
How much work is it to add a line for friction?

**Comment by jtheinen** [comment]
It is doable, I don't have to run any new simulations since I have the data pickled. I just need to transform the forces such that they represent the friction, run the plots and make sure the lay-out is good. 

I will try it this week

**Comment by moorepants** [comment]
Ok, that sounds doable without too much work. I think we should do it and the reviewer should be happy with that. I also makes it clearer!

**Comment by jtheinen** [comment]
![data_no_taildpi600_jasontest](https://github.com/mechmotum/ollie-optimization/assets/90770931/6003590c-8cd2-4674-a7da-481528c0f426)

**Comment by jtheinen** [comment]
![data_no_taildpi600_jasontest_nofric](https://github.com/mechmotum/ollie-optimization/assets/90770931/0a1fc82f-8575-426e-b6ef-333e7e92011d)

**Comment by jtheinen** [comment]
Here you  can see results of adding friction forces. You can see when angle of the board = 0, friction forces of the front foot are the opposite of abduction forces. You can also see that friction occurs during the airtime. 

Let me know if I should change some things before I start plotting all of them

**Comment by moorepants** [comment]
I think adding the two new curves is not too busy and it nicely shows the friction for both feet. I'm surprised to see so much rear foot movement before impact (and thus friction). Also I would have guessed that there would still be front foot friction lifting the board up post impact (nice to see a lot just pre impact).

Is the friction both static and dynamic friction?

I think we should add the curves on all figures.

**Comment by moorepants** [comment]
One thing we could mention in the paper is how the lead foot's shoe is quickly ruined by the friction. I just realized that, unless you are skateboarder, you would never know this.

**Comment by moorepants** [comment]
Also the fact that griptape is applied and doing an ollie without grip tape is significantly more challenging.

**Comment by jtheinen** [comment]
> Also the fact that griptape is applied and doing an ollie without grip tape is significantly more challenging.

I have showed this in my thesis, we could reference to it if we like. I did a penny board without griptape (mu=0.2, with griptape this mu=0.8) this board was able to ollie less high

**Comment by jtheinen** [comment]
> I think adding the two new curves is not too busy and it nicely shows the friction for both feet. I'm surprised to see so much rear foot movement before impact (and thus friction). Also I would have guessed that there would still be front foot friction lifting the board up post impact (nice to see a lot just pre impact).

> Is the friction both static and dynamic friction?

There is not much rear foot motion, if you see the speeds plot, you can see that s1_dot and s2_dot are mostly zero during the pre-pop phase. This means that it consists of mostly static friction. This is logical since the tail of the board is inclined, without static friction the foot would slide down the tail. 

When entering the flight phase you can see clearly that s2_dot increases, which means the front foot is moving. This means that we are dealing with dynamic friction

**Comment by moorepants** [comment]
> I have showed this in my thesis, we could reference to it if we like. I did a penny board without griptape (mu=0.2, with griptape this mu=0.8) this board was able to ollie less high

Reference this in the letter.

**Comment by moorepants** [comment]
> There is not much rear foot motion, if you see the speeds plot, you can see that s1_dot and s2_dot are mostly zero during the pre-pop phase. This means that it consists of mostly static friction. This is logical since the tail of the board is inclined, without static friction the foot would slide down the tail.

Good point, we should probably say this a bit in the paper.

**Comment by jtheinen** [comment]
Figures are added in overleaf, I should check the text and edit captions before closing this issue

**Comment by moorepants** [comment]
> Show already present simulation of the pennyboard without friction, he can see that the ollie will be lower

@jtheinen is this in your thesis? I was looking for it so I could reference it in the letter, but couldn't find it.

**Comment by moorepants** [comment]
Is it appendix figure 29 part a?

**Comment by moorepants** [comment]
I've added descriptions of the friction forces in the "Base Skateboard Optimization" section.
